spec/mailers/user_mailer_spec.rb

```diff
@@ -1205,10 +1205,25 @@
     end
   end
 
+  shared_examples "sends an English email" do
+    it "sends an English email" do
+      expect(I18n).to receive(:with_locale).with("en").and_call_original
+      email = UserMailer.admin_hidden_work_notification(work.id, user.id)
+      expect(email).to have_html_part_content("fan-run and fan-supported archive")
+      expect(email).not_to have_html_part_content("translated footer")
+      expect(email).to have_text_part_content("fan-run and fan-supported archive")
+      expect(email).not_to have_text_part_content("translated footer")
+    end
+  end
+
   describe "obeys the set locale preference feature flag" do
```

Comment:
When a mailer preview is added for admin_hidden_work_notification, its `I18n.with_locale` call will have to be moved outside of `UserMailer.admin_hidden_work_notification` and this test will break.

Proposal: I could remove this test and expand the cucumber test added in this PR to turn the feature flag off again (with the preference still set), send another email and check that it's not translated.